pages/_app.tsx

```diff
@@ -43,22 +44,29 @@ const App: FC<Props> = ({ Component, pageProps }) => {
     }
   }, [])
 
+  const client = new ApolloClient({
+    uri: 'http://localhost:3000/api/graphql',
```

Comment:
I think when you change it for a relative path it works (this way it will also run in production)
```suggestion
    uri: '/api/graphql',
```

Reply:
If it doesn't work this way, I think we need to create an environment variable for it 🤔